### ISSUE TITLE
Crash on video preview [Linux]

### ISSUE THREAD

**Issue by TheChymera** [uTox/uTox#1145]
Using utox `0.16.1` on Gentoo linux, as soon as I try to preview the video chat functionality, I get:

```
chymera@zenbookhost ~ $ utox --debug log_trace
ASAN:DEADLYSIGNAL
=================================================================
==5328==ERROR: AddressSanitizer: SEGV on unknown address 0x000000954958 (pc 0x000000465707 bp 0x7ffe9c7fd980 sp 0x7ffe9c7fd720 T0)
    #0 0x465706 in video_frame (/usr/bin/utox+0x465706)
    #1 0x4d1df6 in utox_message_dispatch (/usr/bin/utox+0x4d1df6)
    #2 0x471b01 in doevent (/usr/bin/utox+0x471b01)
    #3 0x41473b in main (/usr/bin/utox+0x41473b)
    #4 0x7f227b4fd040 in __libc_start_main (/lib64/libc.so.6+0x21040)
    #5 0x415429 in _start (/usr/bin/utox+0x415429)

AddressSanitizer can not provide additional info.
SUMMARY: AddressSanitizer: SEGV (/usr/bin/utox+0x465706) in video_frame
==5328==ABORTING
```

And the program unceremoniously closes. I'm not sure I am using the debug option correctly.

**Comment by endoffile78** [comment]
This might be fixed by #1130.

**Comment by TheChymera** [comment]
yep, fixed!
